Gemfile

```diff
@@ -70,7 +70,7 @@ gem 'rqrcode'
 gem 'ruby-progressbar'
 gem 'ruby-saml'
 gem 'safe_target_blank', '>= 1.0.2'
-gem 'saml_idp', github: '18F/saml_idp', tag: '0.23.9-18f'
+gem 'saml_idp', github: '18F/saml_idp', ref: '573da0f'
```

Comment:
We might need to update this once this PR is merged in the saml_idp repo: https://github.com/18F/saml_idp/pull/156